copilot/download-report/manifest.yml

```
@@ -28,6 +28,7 @@ variables:                    # Pass environment variables as key value pairs.
   # Sentry DSN is OK to be public see: https://docs.sentry.io/product/sentry-basics/dsn-explainer/#dsn-utilization
   SENTRY_DSN: https://6a2623e302e641ba88eabe6675a70ddf@o1432034.ingest.sentry.io/4505390859747328
   FLASK_ENV:  ${COPILOT_ENVIRONMENT_NAME}
+  ACCOUNT_STORE_API_HOST: http://fsd-account-store.dev.pre-award.local:8080
```

This is hard-coded to dev, so would break on other envs. I think this should work instead:
```suggestion
  ACCOUNT_STORE_API_HOST: http://fsd-account-store.${COPILOT_ENVIRONMENT_NAME}.pre-award.local:8080
```

---

Changed as requested.